### PR TITLE
stop auto remembering flags in 2.0

### PR DESCRIPTION
**Pull request by pducks32**
Closes https://trello.com/c/yGsPNDpg

### CODE REVIEW

**Comment by indirect** [comment]
Pretty sure this is already done in the 2-0-dev branch? But we should check. :)

**Comment by arthurnn** [comment]
:+1: . was pretty confused on 1.9 by this behaviour.

**Comment by pducks32** [comment]
Oh well if it was fixed in the dev branch then we can close this. I fixed the errors–alias is a keyword not a method–but its fine.

File: lib/bundler/settings.rb [hunk 1]
```diff
@@ -44,7 +50,7 @@ def set_global(key, value)
     def all
       env_keys = ENV.keys.select { |k| k =~ /BUNDLE_.*/ }
 
-      keys = @global_config.keys | @local_config.keys | env_keys
+      keys = @cuurent_config.keys | @global_config.keys | @local_config.keys | env_keys | DEFAULT_CONFIG.keys
```
Comment (smlance) [inline lib/bundler/settings.rb]: Typo (cuurent -> current)

**Comment by indirect** [comment]
Whoops! Totally my bad, the 2-0-dev branch does not have this. We'll merge it over there and remove the version checks. :)

**Comment by pducks32** [comment]
Oh wait then give me some time to fix this code to make it pass, I only quickly put it together.
Patrick

**Pull request closed without merging**